automation/check-patch.setup.sh

```diff
@@ -12,5 +12,6 @@ export TMP_PROJECT_PATH=$tmp_dir/kubernetes-nmstate
 export ARTIFACTS=${ARTIFACTS-$tmp_dir/artifacts}
 mkdir -p $ARTIFACTS
 
+cp $TMP_PROJECT_PATH/test_logs/e2e/*.log $ARTIFACTS
```

Comment:
This is setup script, it is called before any tests are executed. You have several options:

1. Dump those logs directly to $ARTIFACTS
2. Introduce teardown script
3. Add explicit `cp` to each lane after tests finish

Reply:
Got it. Added the relevant line to providers scripts.